manifests/service.pp

```diff
@@ -34,4 +34,12 @@
     status     => 'service pulp_resource_manager status | grep "node resource_manager"',
   }
 
+  service { 'pulp_streamer':
+    ensure     => running,
+    require    => [Service[mongodb]],
```

Comment:
I'm not on the pulp team, but I would like to request to not force the pulp server to be on the same machine as the mongodb.